src/libraries/System.Security.AccessControl/src/System/Security/Policy/Evidence.cs

```diff
@@ -22,12 +22,12 @@ public sealed partial class Evidence : ICollection, IEnumerable
         public void AddAssembly(object id) { }
         public void AddAssemblyEvidence<T>(T evidence) where T : EvidenceBase { }
         public void AddHostEvidence<T>(T evidence) where T : EvidenceBase { }
-        public T GetAssemblyEvidence<T>() where T : EvidenceBase { return default(T); }
-        public T GetHostEvidence<T>() where T : EvidenceBase { return default(T); }
+        public T? GetAssemblyEvidence<T>() where T : EvidenceBase { return default(T); }
```

Comment:
This might need to go into the nullable annotations breaking change issue? Also, this annotations should be added to the ref as well?

Reply:
I believe the annotation changed because the type moved down to an assembly that's annotated from an assembly which was not annotated.  As such I'm not sure we'd consider it breaking.

I'll make sure to get it added to the ref.